lib/ts/recipe/dashboard/api/getTenantLoginMethodsInfo.ts

```diff
@@ -97,11 +106,9 @@ function normaliseTenantLoginMethodsWithInitConfig(
     if (tenantDetailsFromCore.passwordless.enabled === true) {
         try {
             const thirdpartyPasswordlessRecipe = ThirdPartyPasswordlessRecipe.getInstanceOrThrowError();
-            normalisedTenantLoginMethodsInfo.passwordless.enabled = true;
-            normalisedTenantLoginMethodsInfo.passwordless.contactMethod =
+            normalisedTenantLoginMethodsInfo.thirdPartyPasswordless!.enabled = true;
+            normalisedTenantLoginMethodsInfo.thirdPartyPasswordless!.contactMethod =
```

Comment:
why do you need to have the `!` at the end of `thirdPartyPasswordless`?

Reply:
this is because of bad response type that I've chosen before. Now I've updated with the correct set of types..